docs/external-frameworks-multikueue.md

```
@@ -0,0 +1,82 @@
+metadata:
+  name: kueue-manager-config
+  namespace: kueue-system
+data:
+  multikueue:
+    externalFrameworks:
+      - name: "PipelineRun.v1.tekton.dev"
+```
+
+### Custom Job with Different managedBy Path
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)